src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/ManualRemoveView.tsx

```diff
@@ -141,7 +141,13 @@ export function ManualRemoveView(props: Props) {
                   scanResult={scanResult}
                   isExpanded={index === activeExposureCardKey}
                   isPremiumUser={props.isPremiumUser}
-                  setExpanded={() => setActiveExposureCardKey(index)}
+                  setExpanded={() => {
+                    if (index === activeExposureCardKey) {
+                      setActiveExposureCardKey(-1);
```

Comment:
Q: Why was this a string (`setActiveExposureCardKey("")`) in /user/dashboard/View.tsx#L129 above, but an index (`setActiveExposureCardKey(-1)`) here?

Reply:
The state/indicator can be anything really as long as it's consistent. In view, we are using a composite key in the form of a string b/c we have two types of exposures, this one is more straightforward

Reply:
That confused me as well for a second. I think it would be nice to be consistent and use a key rather than index for both to keep track of the state. Since we are already doing it this way that looks good to me, though.

Reply:
Here we can use either `scanResult.onerep_scan_result_id` or `index` as the state, both are type `number`